op-node/rollup/driver/state.go

```diff
@@ -192,8 +192,21 @@ func (s *Driver) eventLoop() {
 	var sequencerCh <-chan time.Time
 	planSequencerAction := func() {
 		delay := s.sequencer.PlanNextSequencerAction()
+		if delay == 0 {
+			if len(sequencerCh) > 0 {
+				// We want to perform the next action immediately and it's already scheduled so just leave it alone.
+				return
+			}
+			// Otherwise create a channel and trigger it immediately.
```

Comment:
Should probably also stop the timer in this case.

Reply:
Yeah worth while so we don't leave it hanging around only to ignore its result.